source/_docs/ecosystem/caddy.markdown

```diff
@@ -10,12 +10,40 @@ footer: true
 ---
 
 Configure [Caddy Server](https://caddyserver.com/) for use as a reverse proxy to Home Assistant.
+## Install
 
+<p class='note warning'>
+It can potentially be dangerous to send a command straight to bash please make sure you trust the source before running that type of commands, to check what this command does head over to [getcaddy.com](https://getcaddy.com/)
+</p>
+
+Once you are ready to proceed with the installer type:
+```
+curl https://getcaddy.com | bash -s personal
 ```
-hass.example.org {
-    proxy / localhost:8123 {
+
+ 
+Then create a file named ``Caddyfile``:
+```
+hass.yourdomain.org {
+    proxy / yourhassinternalip:8123 {
         websocket
         transparent
     }
 }
 ```
+
+Make sure to forward ports: `80` and `443` on your router so that the SSL certificates can be generated
```

Comment:
Maybe clarify that those steps (like forwarding ports and setting the flags) are only when users who want to automatically obtain Let's Encrypt certificates for HTTPS?

Reply:
Don't you still need to forward 443 if you are going to use HTTPS at all

Reply:
I use `proxy / localhost:8123` in my caddyfile and it works fine. Is there a reason to use the ip instead?

Reply:
That would only work if Caddy is installed on the same system. 
That might indeed be the case most of the time. However, when thinking like that, most users are using HassOS as well and won't install Caddy this way either.